inc/update.php

```diff
@@ -514,217 +500,6 @@ function largo_update_custom_less_variables() {
 	}
 }
 
-/**
- * Checks for use of deprecated widgets and posts an alert
- */
-function largo_check_deprecated_widgets() {
```

Comment:
We will need to keep this around in the event that someone skips a version of Largo. 

And we should probably keep the deprecation code framework portion of the update tools (and corresponding tests) around in any case, because it was a pain to write the first time and we'll probably deprecate widgets in the future, and won't want to rewrite all this.